packages/mux-video/package.json

```diff
@@ -19,7 +19,7 @@
     "build": "npm-run-all build:snowpack build:types"
   },
   "dependencies": {
-    "hls.js": "^1.0.9",
+    "hls.js": "1.0.11",
```

Comment:
Calling out a change here. An alternate would be to specify:

```
"hls.js": "^1.0.11",
```

However, I think we *do* want to pin to a precise version to avoid pulling in versions of Hls.js that have an accidental regression.

The tradeoff will be that we have to actively keep this dependency up-to-date when Hls.js gets new patch releases.